modules/navier_stokes/test/tests/finite_volume/wcns/channel-flow/2d-transient-physics-energy.i

```diff
@@ -0,0 +1,139 @@
+l = 10
+
+# Artificial fluid properties
+# For a real case, use a GeneralFluidFunctorProperties and a viscosity rampdown
+# or initialize very well!
+k = 1
+cp = 1000
+mu = 1e2
+
+# Operating conditions
+inlet_temp = 300
+outlet_pressure = 1e5
+inlet_v = 0.001
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = ${l}
+    ymin = 0
+    ymax = 1
+    nx = 20
+    ny = 10
+  []
+[]
+
+[Physics]
+  [NavierStokes]
```

Comment:
Personally I'm not a fan of the nesting under `NavierStokes` - seems redundant with `WCNSFVFlow`, but it's ok if you feel strongly about it.

Reply:
I'm confused - this input file has `Physics/NavierStokes/WCNSFVFlow/*`, but I don't see that syntax registered, only `Physics/NavierStokes/Flow/*`.

Reply:
Input file aintgotno test!?

Reply:
probably so

Reply:
it's a duplicate of the other one checked in. I ll remove it now

Reply:
deleted from history

Reply:
But I realize this is the trend you've set with your previous physics - I'll live.